main.go

```diff
@@ -150,17 +150,35 @@ var (
 		// They are there, so include them.
 		newPerThreadGaugeMetric("decoder", "packet_size_avg", "", "avg_pkt_size"),
 		newPerThreadGaugeMetric("decoder", "packet_size_max", "", "max_pkt_size"),
+		newPerThreadGaugeMetric("decoder", "mac_address_src_max", "", "max_mac_addrs_src"),
+		newPerThreadGaugeMetric("decoder", "mac_address_dst_max", "", "max_mac_addrs_dst"),
 
 		newPerThreadCounterMetric("decoder", "too_many_layers_total", "", "too_many_layers"),
 	}
 
+	// From: .thread.decoder.event.afpacket
+	// New in 8.0.0: d78f2c9a4e2b59f44daeddff098915084493d08d
+	perThreadDecoderEventAFPacketMetrics = []metricInfo{
+		newPerThreadCounterMetric("decoder_event", "afpacket_trunc_pkt", "", "trunc_pkt").Optional(),
```

Comment:
```suggestion
		newPerThreadCounterMetric("decoder_event", "afpacket_truncated_packets_total", "", "trunc_pkt"),
```
Mostly to align with the other decoder packet counter metrics above (udp_packets_total) and dropping `Optional()` as it seems if  `afpacket` is within `decoder.event`, can expect `trunc_pkt` to be included.

---

Aside, seems a bit unfortunate the metric is not part of the `capture.afpacket` object as the event is raised in `source-af-packet.c`, but not sure it's worth an upstream ticket to move or also include it there.

@jlucovsky / @jasonish, thoughts? Nothing urgent or to do, just some feedback / food for thought, for the `afpacket.trunc_pkt` stat.

Reply:
Ok for the rename. I also removed the Optional()